applications/HDF5Application/custom_processes/hdf5_xdmf_connectivities_writer_process.cpp

```diff
@@ -18,7 +19,7 @@ XdmfConnectivitiesWriterProcess::XdmfConnectivitiesWriterProcess(const std::stri
                 "file_access_mode": "read_write"
             })");
     file_params["file_name"].SetString(rFileName);
-    mpFile = FileSerial::Pointer(new FileSerial(file_params));
+    mpFile = File::Pointer(new File(ParallelEnvironment::GetDefaultDataCommunicator(), file_params));
```

Comment:
this should come from outside

Reply:
Since xdmf generation requires the indices which is obtained in serial only, this process is run with the serial data communicator.